src/utils/randomBetween.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Get random number between two numbers.
+ * @param min Min number.
+ * @param max Max number.
+ */
```

Comment:
I think it's worth mentioning in the docs:

1. Is a random number an integer or float?
2. Could a random value to be equal to `min` or `max`?

FYI, Math.random() can not return 1, therefore `Math.random() * (max - min) + min` could never be equal to `max`